src/views/frontend/widget/record_list.tpl

```diff
@@ -33,3 +33,31 @@
         </form>
     </ff-record>
 </ff-record-list>
+
+
+<script>
+    function scrollToCallback (selector, config) {
+        debugger
+        const defaultConfig = {
+            behavior: 'smooth',
+            top: 0,
+            left: 0
+        };
+        const mergedConfig = Object.assign({}, defaultConfig, config);
+
+        if (selector) {
+            const element = document.querySelector(selector);
+            !element ? console.error('Invalid selector', selector) : element.scrollIntoView(mergedConfig);
+        } else {
+            window.scrollTo(mergedConfig);
+        }
+    }
+
+    document.addEventListener('ffReady', function (ff) {
+        factfinder.communication.EventAggregator.addBeforeDispatchingCallback(function (event) {
+            if (event.type === 'search' || event.type === 'paging') {
+                event.success = factfinder.common.concatFunctions(event.success, scrollToCallback());
```

Comment:
despite the fact that is 1:1 copied from the M2 [PR ](https://github.com/FACT-Finder-Web-Components/magento2-module/pull/365), it is copied wrong. This will work different than in Magento2
Can you tell what is the difference?

Reply:
fixed that